### DISCUSSION TITLE
part-ii/2020-2/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#120] [Question comments]
Part II 2020 Algebraic Geometry: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2020-2/

**Comment by eha1234** [comment]
(iv) Got this to be d=3. (d=1 doesn't exist by Valency Theorem, d=2 doesn't exist because the canonical is an embedding, d=3 take projection from point on curve)

**Reply by artie2000** [reply]
Alternative for d=1: The divisor $D=\phi^\star(\infty)$ has degree 1 and dimension 2; this satisfies the embedding criterion, so $C\cong\mathbb{P}^1$.
